app/services/listing_index_service/search/commons.rb

```diff
@@ -0,0 +1,85 @@
+module ListingIndexService::Search::Commons
```

Comment:
I guess this module could have a more descriptive name. All the methods seem to have something to do about converting models to hashes, so could the module be `Converters` or something like that?